packages/fluentui/accessibility/src/behaviors/Icon/indicatorBehavior.ts

```diff
@@ -0,0 +1,22 @@
+import { Accessibility, AccessibilityAttributes } from '../../types';
+
+/**
+ * @description
+ * Indicator is usually only visual representation and therefore is hidden from screen readers, unless 'aria-label' property is provided.
+ *
+ * @specification
+ * Adds role='img'.
+ * Adds attribute 'aria-hidden=true', if there is no 'aria-label' property provided.
+ */
+const indicatorBehavior: Accessibility<IndicatorBehaviorProps> = props => ({
+  attributes: {
+    root: {
+      role: 'img',
+      'aria-hidden': props['aria-label'] ? undefined : 'true'
```

Comment:
```suggestion
      'aria-hidden': true,
```

Description should be also changed I believe